src/Client.php

```diff
@@ -163,7 +170,7 @@ public function subscribeToList($listId, $email, $mergeVars = [], $doubleOptin =
                 $requestData['interests'] = $interests;
             }
 
-            $response = $this->post($endpoint, $requestData);
+            $response = $this->put($endpoint . '/' . $this->getSubscriberHash($email), $requestData);
```

Comment:
This response could probably be `null` as well...

Reply:
done